crates/base-db/src/input.rs

```diff
@@ -6,17 +6,23 @@
 //! actual IO. See `vfs` and `project_model` in the `rust-analyzer` crate for how
 //! actual IO is done and lowered to input.
 
+use std::hash::BuildHasherDefault;
```

Comment:
I think this could just be a [FxBuildHasher](https://docs.rs/rustc-hash/latest/rustc_hash/struct.FxBuildHasher.html)?